pkg/controller/import-controller.go

```diff
@@ -261,6 +276,9 @@ func (r *ImportReconciler) updatePvcFromPod(pvc *corev1.PersistentVolumeClaim, p
 				return err
 			}
 		}
+		anno[AnnRunningCondition] = "false"
+		anno[AnnLastTerminationMessage] = "Creating scratch space"
+		anno[AnnLastTerminationReason] = creatingScratch
```

Comment:
These are set when the pod is pending and needs scratch space.  But when the pod is no longer pending, it will display the real message/reason.  Is that okay?

Also, do any import types start out with scratch space?  If so it will never make sense to do this.

Reply:
There are certain import types we know will have scratch space, in particular registry import.